components/summary-table.js

```diff
@@ -0,0 +1,54 @@
+// node_modules
+import PropTypes from "prop-types";
+import Link from "next/link";
+// components
+import { DataGridContainer } from "./data-grid";
+import SortableGrid from "./sortable-grid";
+
+/**
+ * Display a sortable table of the given data.
+ */
+export default function SummaryTable({ data, assembly }) {
+  const summaryColumns = [
+    {
+      id: "chrom_location",
+      title: "Chromosome location",
+      display: ({ source }) => {
+        const url = `/search?regions=${source.chrom_location}&genome=${assembly}`;
+        return <Link href={url}>{source.chrom_location}</Link>;
+      },
+    },
+    {
+      id: "rsids",
+      title: "dbSNP IDs",
+      display: ({ source }) => `${source.rsids.join(", ")}`,
+    },
+    {
+      id: "rank",
+      title: "Rank",
+    },
+    {
+      id: "score",
+      title: "Score",
+    },
+  ];
+  const initialSort = {
+    columnId: "rank",
+  };
```

Comment:
I would move `summaryColumns` and `initialSort` outside the `SummaryTable` component so that they don’t get set and set again to the same value on every render.

Reply:
I can move initialSort outside the SummaryTable, but summaryColumns needs to know assembly value, which it get from SummaryTable. Any better approach?

Reply:
I would pass the assembly in the meta object.